### PR TITLE
Feat/PROTECT-1720 - LLD Nano onboarding to include a screen about recover

### PR DESCRIPTION
**Pull request by nparigi-ledger**
### 📝 Description

Add an upsell recover message after classic onboarding

### ❓ Context

- **Impacted projects**: `` live-common, live-desktop
- **Linked resource(s)**: `` https://ledgerhq.atlassian.net/browse/PROTECT-1720
- 
### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by jules-grenier-ledger — APPROVED** [review]
LGTM

File: libs/ledger-live-common/src/hooks/recoverFeatueFlag.ts [hunk 1]
```diff
@@ -46,3 +46,12 @@ export function useAlreadySubscribedURI(
 
   return useReplacedURI(uri, id);
 }
+
+export function useUpsellURI(
+  servicesConfig: Feature<any> | null
```
Comment (gre) [inline libs/ledger-live-common/src/hooks/recoverFeatueFlag.ts:51]: `any` is no longer allowed on LLD project, you can possibly use the correct type OR unknown (which will requires you to do a lot of typecheck in the JS if you use unknown)

**Review by gre — CHANGES_REQUESTED** [review]
👋  we have recently merged https://github.com/LedgerHQ/ledger-live/pull/3455 in develop branch which reinforces the TypeScript checks, mostly related to modals and coin families. Could you rebase and check that everything still pass?

Thanks,
gre.

**Comment by Justkant** [comment]
done in #3644

**Pull request closed without merging**